SuffolkLITLab/courtformsonline.org
# dir restructure

Github actions isnt currently picking up the new workflows we added in the last couple PRs.  I think this is because we have an extra directory that the project is nested in.  Either way we will want to clean this up and move the project contents to the root dir.

This PR moves the contents of the courtformsonline.org dir to its parent.

QA Steps:
- Note that the build check is now running in checks for this PR.
- There should be no other noticeable changes to the app
